src/spikeinterface/postprocessing/amplitude_scalings.py

```diff
@@ -427,13 +431,13 @@ def _are_unit_indices_overlapping(unit_inds_to_channel_indices, i, j):
     bool
         True if the unit indices i and j are overlapping, False otherwise
     """
-    if len(np.intersect1d(unit_inds_to_channel_indices[i], unit_inds_to_channel_indices[j])) > 0:
+    if np.sum(np.logical_and(sparsity_mask[i], sparsity_mask[j])) > 0:
         return True
     else:
         return False
 
```

Comment:
I thought for sure that doing 
`return np.sum(np.logical_and(sparsity_mask[i], sparsity_mask[j])) >0` was going to be faster, but when I ran `timeit` they were always within 2 microseconds of each other and would take turns being the faster.